internal/core/config.go

```diff
@@ -39,26 +40,24 @@ import (
 // cluster. It is instantiated from YAML and then transformed into type
 // Cluster during the startup phase.
 type ClusterConfiguration struct {
-	AvailabilityZones []limes.AvailabilityZone `yaml:"availability_zones"`
-	CatalogURL        string                   `yaml:"catalog_url"`
-	Discovery         DiscoveryConfiguration   `yaml:"discovery"`
-	Services          []ServiceConfiguration   `yaml:"services"`
-	Capacitors        []CapacitorConfiguration `yaml:"capacitors"`
-	// ^ Sorry for the stupid pun. Not.
+	AvailabilityZones        []limes.AvailabilityZone         `yaml:"availability_zones"`
+	CatalogURL               string                           `yaml:"catalog_url"`
+	Discovery                DiscoveryConfiguration           `yaml:"discovery"`
+	Liquids                  []LiquidConfiguration            `yaml:"liquids"`
```

Comment:
I messed up and forgot to mention one important thing in the ticket:

```suggestion
	Liquids                  map[db.ServiceType]LiquidConfiguration            `yaml:"liquids"`
```

I would like for this to be a map instead of a list, because that would enable maybe refactoring the Helm values to reduce the templating a bit. (With lists, everything needs to be in a single values.yaml because there is no good way to merge bits and pieces from multiple values.yaml files together. The only merge strategy for lists is to concatenate them, so a later values.yaml file cannot modify a LiquidConfiguration added by an earlier values.yaml file.)